packages/desktop-client/src/components/budget/rollover/HoldTooltip.tsx

```diff
@@ -29,7 +29,7 @@ export default function HoldTooltip({ onSubmit, onClose }: HoldTooltipProps) {
   useEffect(() => {
     (async () => {
       const node = await spreadsheet.get(sheetName, 'to-budget');
-      setAmount(integerToCurrency(Math.max(node.value, 0)));
+      setAmount(integerToCurrency(Math.max(node.value as number, 0)));
```

Comment:
spreadsheet value type can be string or number. We expect it to always be a number in this context, so we have to tell it it's a number.